src/lib/cat_superhero.svelte

```diff
@@ -4,46 +4,107 @@
+      imageName: 'trap.png',
+      imageAlt: '',
+      extraContent:
+        'Trap between 4 pm to 11.30 pm the evening before your appointment. Set food on some tin foil and place under the trap (not inside). Keep an eye on the trap – never leave it unattended – and wait for kitty to go inside. Be patient! <a href="https://www.youtube.com/watch?v=IarsSTLUdTw">Trap-Neuter-Return</a> is the first step towards a happier and healthier life for your community cat friend.',
+      resourceTexts: ['TNR'],
+      resourceLinks: ['https://www.youtube.com/watch?v=IarsSTLUdTw']
     },
     {
       title: 'Hold',
-      text: "Hold cat overnight until next day's surgery"
+      text: 'Hold the cat overnight until surgery the next day',
+      imageName: 'hold.png',
+      imageAlt: '',
+      extraContent:
+        'Congrats, you got the cat! Cover the trap and keep it in a warm, quiet place overnight (like a bathroom or a heated garage). Check the cat once an hour, every hour, until bedtime. Here are some <a href="https://fixnation.org/about-tnr/pre-post-surgery-instructions/">more pre-op tips!</a>',
+      resourceTexts: ['Pre-op Tips'],
+      resourceLinks: ['https://fixnation.org/about-tnr/pre-post-surgery-instructions/']
     },
     {
       title: 'Fix',
-      text: 'Fix cat for free at FixNation'
+      text: 'Take the cat to get fixed (for free!).',
+      imageName: 'fix.png',
+      imageAlt: '',
+      extraContent:
+        'Feed kittens a teapoon of wet food at 6 am on surgery day; no food or water for older cats. Cover the car seats with plastic and securely place the traps on top. Arrive at FixNation between 7-8.30 am, leave kitty in car and wait in line to check-in. (Here’s a <a href="https://www.facebook.com/LuxePaws/videos/3379488712127276/">sneak peek of the check-in process!</a>) FixNation will spay/neuter the cat and provide vaccines, flea treatment, fluids, pain medication and an ear tip. Pick-up from 3 - 4.30 pm.',
+      resourceTexts: ['Check-in Process'],
+      resourceLinks: ['https://www.facebook.com/LuxePaws/videos/3379488712127276/']
     },
     {
-      title: 'Hold',
-      text: 'Hold cat overnight for recovery'
+      title: 'Watch',
+      text: 'Keep the cat one more night for recovery.',
+      imageName: 'watch.png',
+      imageAlt: '',
+      extraContent:
+        'Take kitty home in the covered trap and keep in a warm, quiet place overnight once again. Feed a ¼ can of wet food about six hours post-op (check paperwork for exact time). Check kitty once an hour, every hour, until bedtime. Here are some <a href="https://www.alleycat.org/community-cat-care/post-surgery-care/">more post-op tips!</a>',
+      resourceTexts: ['Post-ops tips'],
+      resourceLinks: ['https://www.alleycat.org/community-cat-care/post-surgery-care/']
     },
     {
       title: 'Return',
-      text: 'Return cat to where it was trapped'
+      text: 'Return the cat to where it was trapped',
```

Comment:
```suggestion
      text: 'Return the cat to where it was trapped.',
```
To match the formatting on the Figma.